roles/confluent.kafka/tasks/main.yml

```diff
@@ -1,139 +1,139 @@
 - name: broker group
   group:
-    name: "{{kafka.broker.group}}"
+    name: "{{kafka.group}}"
 - name: broker user
   user:
-    name: "{{kafka.broker.user}}"
+    name: "{{kafka.user}}"
     comment: "Kafka User"
     system: yes
-    group: "{{kafka.broker.group}}"
+    group: "{{kafka.group}}"
 - name: Change keytabs ownership
   file:
-    path: "{{kafka.broker.keytab}}"
-    group: "{{kafka.broker.group}}"
+    path: "{{kerberos.keytab_dir}}"
+    group: "{{kafka.group}}"
```

Comment:
Recommend leaving in 'broker' as kafka is not a component but the platform name.